pkg/pillar/nireconciler/linux_reconciler.go

```diff
@@ -1270,12 +1277,51 @@ func (r *LinuxNIReconciler) getNIRouteInfo(niID uuid.UUID) (routes []types.IPRou
 				}
 			}
 		}
+		var ipVer types.AddressType
+		switch route.Type() {
+		case generic.IPv4RouteTypename:
+			ipVer = types.AddressTypeIPV4
+		case generic.IPv6RouteTypename:
+			ipVer = types.AddressTypeIPV6
```

Comment:
I see the `Type()` function may also return `generic items.UnsupportedRouteTypename`. Shouldn't we add it here?

Reply:
We will leave the zero value `types.AddressTypeNone` in that case which is OK I think.